website/pages/docs/concepts/seal.mdx

```diff
@@ -89,75 +89,159 @@ Recovery Keys and are used for these operations.
+be migrated as the recovery keys for the KMS seal.
 
-```
-$ vault operator unseal -migrate
-```
+4. Standby nodes can join the active node's cluster using the new seal blocks.
+Note that by the time standby nodes join the cluster, the migration process is
+already complete. Hence, the standby nodes will be auto-unsealed through the KMS
+key and will not require an explicit unseal command.
 
-### Migration From Auto Unseal to Shamir
+#### When using Integrated Storage
+
+Integrated Storage is implemented using the [Raft Consensus Algorithm][raft] and
+would require a quorum of nodes to be fully operational.
+
+1. Take your server cluster offline.
+
+2. Add the desired KMS [seal block](/docs/configuration/seal) to the
+configuration files of quorum number of nodes in the cluster.
+
+3. Bring the quorum number of nodes back up. On each of these nodes, run the
+unseal command by supplying the `-migrate` flag for each unseal key. The Shamir
+unseal keys supplied here will be migrated as the recovery keys for the KMS
+seal. Raft protocol will then elect a leader node and this leader node will
+complete the seal migration process. If a leadership change happens after the
+seal migration is complete, the new leader will detect the fact and does not
+repeat the seal migration.
+
+4. Any new node joining to the cluster at this point will not require manual
+running of the unseal command. The join will ensure auto-unsealing of the node
+through KMS key.
+
+### Migration From KMS Seal to Shamir Seal
 
 ~> **NOTE**: Migration to Shamir seal is not currently supported when using
 Vault Enterprise. We plan to support this officially in a future release.
 
-To migrate from Auto Unseal to Shamir keys, take your server cluster offline and update
-the [seal configuration](/docs/configuration/seal) and add `disabled = "true"`
-to the seal block. This allows the migration to use this information to decrypt the key
-but will not unseal Vault. When you bring your server back up, run the unseal process
-with the `-migrate` flag and use the Recovery Keys to perform the migration. All unseal
-commands must specify the `-migrate` flag. Once the required threshold of recovery keys
-are entered, the recovery keys will be migrated to be used as unseal keys.
+#### When **not** using Integrated Storage
+
+1. Take your server cluster offline.
+
+2. Add `disabled = "true"` to the old KMS [seal block](/docs/configuration/seal)
+in the config file of one of the nodes. Shamir seal does not require an explicit
+seal block.
+
+3. Bring this one node back up and run the unseal command using the recovery
+keys of the KMS seal along with supplying the `-migrate` flag for each recovery
+key. This node will be the active node and completes the seal migration process.
+The recovery keys supplied here will be migrated as the unseal keys for the
+Shamir seal.
 
-### Migration From Auto Unseal to Auto Unseal
+4. Standby nodes can join the active node's cluster without the KMS seal blocks.
```

Comment:
I have the same question here that I did on the corresponding step earlier (basically, can you just re-start them).